lib/wicked_pdf/wicked_pdf_helper.rb

```diff
@@ -95,7 +95,7 @@ def set_protocol(source)
       protocol = WickedPdf.config[:default_protocol] || 'http'
       if source[0, 2] == '//'
         source = [protocol, ':', source].join
-      elsif !source[0, 7].include?('://')
```

Comment:
Imagine you have a variable named source, defined as follows:

```source = "https://cdn.example.com/assets/application_fha2215asfahi21f1.css"```

What would ```source[0,7]``` be? In this case, "https:/", which would not include "://", and so we'd get an asset like:

```"http://https://cdn.example.com/assets/application_fha2215asfahi21f1.css"```